auth/user_mgt.go

```diff
@@ -256,6 +256,25 @@ func (c *Client) GetUserByEmail(ctx context.Context, email string) (*UserRecord,
 	return c.getUser(ctx, request)
 }
 
+// GetUsersByEmail gets the users data corresponding to the specified emails.
+func (c *Client) GetUsersByEmail(ctx context.Context, emails []string) ([]*UserRecord, error) {
+	var userRecords []*UserRecord
+	for _, email := range emails {
+		if err := validateEmail(email); err != nil {
+			return nil, err
+		}
+		request := &identitytoolkit.IdentitytoolkitRelyingpartyGetAccountInfoRequest{
+			Email: []string{email},
```

Comment:
You can pass the entire slice of emails to the request and get all the account info back at once.

Reply:
I am using the existing function https://github.com/firebase/firebase-admin-go/blob/master/auth/user_mgt.go#L642 whose return type is *UserRecord. 
As per what you suggested, probably I would need to create a separate function which returns []*UserRecord. Let me know your thoughts. Thanks.

Reply:
Yes, I think you would have to do that. I am unsure how you would have to format the errors though. Maybe someone from the Firebase team can chime in here?